src/game/configuration/settings.ts

```diff
@@ -9,7 +9,19 @@ export const collisionPointOnBallClass = "bba-collision-point-on-ball"
 export type StartOptions = {
   withScoreboard: boolean
   initialBallSpeed: "low" | "middle" | "high" | "superHigh"
-  sound: boolean
+  sound: boolean,
+  visualizeBlocks: boolean,
+  controlMode: "normal" | "mouse"
+}
+
+export function disableScoreboardIfDebug(options: StartOptions): StartOptions {
```

Comment:
[Q] Is it possible to remove this function and make the scoreboard disabeld for the debug mode within `sendMessageToIsolatedWorldOnActiveTabForTest()` in `popup.tsx`? I'd like to hear your opnion :)

Reply:
👍  Indeed. I'm a little worried that'd make `popup.tsx` complex, but more user-friendly!

Reply:
It seems I mistook your comment wrong 😅 . But https://github.com/canalun/brick-break-anywhere/pull/15/commits/2614729ca72b29736c672db23768156bb82b9fc0 would surpass your expectation!